app/src/main/res/layout/fragment_lots.xml

```diff
@@ -59,5 +59,21 @@
         app:layout_constraintTop_toBottomOf="@id/text_header_lot_list">
     </androidx.recyclerview.widget.RecyclerView>
 
-
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/fab"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:clickable="true"
+        android:contentDescription="@string/submit"
+        android:focusable="true"
+        android:focusableInTouchMode="true"
```

Comment:
"If this is true for a view, that view can gain focus when clicked on"
I think that's why in the second time when I touch the fab view, it works. 
If you have doubts about using an android attribute on xml, search here: https://developer.android.com/reference/android/view/View

I think you should remove this.

Reply:
You should remove focusableInTouchMode
When I run the app, it does not work fab view (for first time).